app/src/main/java/com/example/caregiver/ProfileInfo.java

```diff
@@ -251,8 +269,18 @@ public void onClick(View v) {
         }
     };
 
-    public static class MainActivity extends AppCompatActivity {
+    /**
+     * Function to logout
+     */
+    private View.OnClickListener logOutListener = new View.OnClickListener() {
+        @Override
+        public void onClick(View v) {
+            Intent i = new Intent(v.getContext(), Login.class);
+            startActivity(i);
+        }
+    };
 
+    public static class MainActivity extends AppCompatActivity {
         @Override
         protected void onCreate(Bundle savedInstanceState) {
             super.onCreate(savedInstanceState);
```

Comment:
What's the purpose of this snippet in ProfileInfo?

Reply:
Good point. I have no idea lol why I incude it in there. Will remove it.